plugins/worker/worker.js

```diff
@@ -123,13 +123,17 @@ async function invoke() {
         scheduler.on('poll', () => logger.info('queueWorker->scheduler polling'));
         scheduler.on('master', state => logger.info(`queueWorker->scheduler became master ${state}`));
         scheduler.on('error', error => logger.info(`queueWorker->scheduler error >> ${error}`));
-        scheduler.on('working_timestamp', timestamp =>
```

Comment:
old code seems to be listening to wrong event, i can't not find any log for `working_timestamp` and `transferred_job`